rust/src/nfs/nfs.rs

```diff
@@ -1566,6 +1607,18 @@ pub extern "C" fn rs_nfs_state_get_tx_detect_state(
     }
 }
 
+pub extern "C" fn rs_nfs_udp_state_get_event_info_by_id(_event_id: std::os::raw::c_int,
+                                              event_name: *mut *const std::os::raw::c_char,
+                                              event_type: *mut AppLayerEventType)
+                                             -> i8
+{
+    unsafe {
+        *event_name = b"NFS UDP event name (generic)\0".as_ptr() as *const std::os::raw::c_char;
+        *event_type = APP_LAYER_EVENT_TYPE_TRANSACTION;
+    }
+    0
+}
+
```

Comment:
Move new fn addition to separate commit

Reply:
I actually think the whole event logic is unused in nfs, so lets not move it over